garage/tf/policies/continuous_mlp_policy.py

```diff
@@ -61,6 +60,7 @@ def __init__(self,
         self._hidden_nonlinearity = hidden_nonlinearity
         self._output_nonlinearity = output_nonlinearity
         self._batch_norm = bn
+        self._q_network_name = "q_network"
```

Comment:
Isn't this a policy network?

Reply:
It's mentioned [here](https://github.com/rlworkgroup/garage/blob/99324f0d215631ac09f2c1f89123520b864462ea/garage/tf/policies/continuous_mlp_policy.py#L67) that a Q Network is built, and since this scope is applied to the output of that network, that's why I decided to use that name, but I will change it to "policy_network" if it's more correct.